model/vfs/vfsswift/impl_v2.go

```diff
@@ -399,6 +383,23 @@ func (sfs *swiftVFSV2) destroyFileVersions(objName string) error {
 	return nil
 }
 
+func (sfs *swiftVFSV2) EnsureErased(journal vfs.TrashJournal) error {
+	// No lock needed
+	_, err := sfs.c.BulkDelete(sfs.container, journal.ObjectNames)
+	if err == swift.Forbidden {
+		sfs.log.Infof("EnsureErased failed on BulkDelete: %s", err)
```

Comment:
Info is probably too low for a delete error implying an inconsistency between couch and swift.